### PR TITLE
Issues

### LINKED ISSUE #1: Migrate yarn to 3.x
- [x] Give `yarn install` command in the command line to generate new yarn.lock file.

### PR DESCRIPTION
**Pull request by UmurErgenay**
I migrated to yarn 3.x.x version, and added the ".yarn" folder to gitignore.

resolves #1

### CODE REVIEW

**Comment by UmurErgenay** [comment]
[new yarn.lock file generated] as well

**Review by emreliman — COMMENTED** [review]
✔

File: src/components/TableComponent.vue [hunk 1]
```diff
@@ -34,8 +34,8 @@ type HeaderCellDetails = {
 }
```
Comment (ismailarilik) [inline src/components/TableComponent.vue:34]: It seems that this doesn't belong here.

File: src/table.json [hunk 2]
```diff
@@ -333,7 +333,6 @@
               null,
```
Comment (ismailarilik) [inline src/table.json:333]: It seems that this doesn't belong here.

**Review by ismailarilik — CHANGES_REQUESTED** [review]

**Pull request merged**